Performance fix

developerpeachy opened the pull request:
- Added Webpack to minify the js. 
- did an audit test but it wasn't very accurate, so to test we should deploy this on a staging branch? 
- adjusted some images, but still need to fix the company logos. 

- may need to look at how we can compress the font since that takes the longest to load (shocking right?)

hboon commented:
> * did an audit test but it wasn't very accurate, so to test we should deploy this on a staging branch?

Wasn't there a staging site earlier? Does that still working?

developerpeachy commented:
- Footer is updated with links that now work in between pages.
- Twitter link is updated to the correct handle. 
- Removed a lot of unnecessary components/pages that we didn't really need. 
- Removed some images that weren't used. 
- Added Webpack to ease load times.
